index.html

```diff
@@ -0,0 +1,179 @@
+<!DOCTYPE html>
+<html lang="ru">
+    <head>
+        <title>Шаблоны</title>
+        <meta charset="UTF-8">
+        <link rel="stylesheet" href="./fonts/fonts.css">
+        <link rel="stylesheet" href="./font-awesome-4.7.0/css/font-awesome.min.css">
+        <link rel="stylesheet" href="./style/normalize.css"/>
+        <link rel="stylesheet" href="./style/header/header.css"/>
+        <link rel="stylesheet" href="./style/header/menu/menu.css"/>
+        <link rel="stylesheet" href="./style/header/logo/logo.css"/>
+        <link rel="stylesheet" href="./style/body/body.css"/>
+        <link rel="stylesheet" href="./style/body/navigation/navigation.css"/>
+        <link rel="stylesheet" href="./style/body/navigation/button/button.css"/>
+        <link rel="stylesheet" href="./style/body/navigation/button/_style/button_style_bordered.css"/>
+        <link rel="stylesheet" href="./style/body/templates-selector/templates-selector.css"/>
+        <link rel="stylesheet" href="./style/body/templates-selector/__header/templates-selector__header.css"/>
+        <link rel="stylesheet" href="./style/body/templates-selector/__header/search-box/search-box.css"/>
+        <link rel="stylesheet" href="./style/body/templates-selector/__header/search-box/__input/search-box__input.css"/>
+        <link rel="stylesheet" href="./style/body/templates-selector/wrapper/wrapper.css"/>
+        <link rel="stylesheet" href="./style/body/templates-selector/edit-buttons/edit-buttons.css"/>
+        <link rel="stylesheet" href="./style/body/templates-selector/template-item/template-item.css"/>
+        <link rel="stylesheet" href="./style/body/templates-selector/template-item/__text/template-item__text.css"/>
+        <link rel="stylesheet" href="./style/body/templates-selector/template-item/__counters/template-item__counters.css"/>
+        <link rel="stylesheet" href="./style/body/scroll-panel/scroll-panel.css"/>
+        <link rel="stylesheet" href="./style/footer/footer.css"/>
+    </head>
+    <body class="body">
+        <header class="header">
+            <a class="logo" href=""></a>
+            <ul class="menu">
+                <li>
+                    <a href="#user">
+                        <i class="fa fa-user" aria-hidden="true"></i>
+                        <span>Привет, Админ</span>
+                        <i class="fa fa-caret-down" aria-hidden="true"></i>
+                    </a>
+                </li>
+                <li><a href="#about">О компании</a></li>
+            </ul>
+        </header>
+
+        <div class="container">
+            <ul class="navigation">
```

Comment:
It makes sense to create parent <div class=left-column> element for <ul class="menu">element and move the following styles for it:
     min-width: 220px; 
     box-sizing: border-box; 
     border-bottom-style: solid; 
     border-right-style: solid; 
     border-width: 4px; 
     padding: 10px; 

It will be more flexible. <ul class="menu"> should be responsible only for displaying menu items. But defining of width, right- border - it is a responsibility of parent element. Imagine, if we need to add one more element in the left column. In this case we should modify styles for <ul class="menu">.